api/src/main/java/org/pmiops/workbench/exfiltration/EgressRemediationService.java

```diff
@@ -280,6 +290,18 @@ protected void disableUser(DbUser user) {
     stopUserRuntimesAndApps(user.getUsername());
   }
 
+  protected void disableUserVwbDataAccess(DbUser user) {
+    userService.updateUserWithRetries(
```

Comment:
maybe just call disableUser here instead of duplicating the code?

Reply:
i will be adding logic in my next PR here. So I probably will leave it as it and come back to modify soon